db/seeds.rb

```diff
@@ -6,7 +6,11 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-topic_descriptions = %w[Music\ Sports\ Videogames\ Gardening\ Travelling\ Space]
-topic_descriptions do |description|
+topic_descriptions = %w[Music Sports Videogames Gardening Travelling Space]
+topic_descriptions.each do |description|
   topic = Topic.find_or_create_by!(description: description)
```

Comment:
do you need to assign the result to that `topic` variable?
what about `Topic.create(description: description)` without `!` so errors don't raise